integration/kubernetes/k8s-number-cpus.bats

```diff
@@ -31,9 +31,11 @@ setup() {
 		# Get number of cpus
 		number_cpus=$(kubectl exec pod/"$pod_name" -c "$container_name" \
 			-- sh -c "$num_cpus_cmd")
-		# Verify number of cpus
-		[ "$number_cpus" -le "$max_number_cpus" ]
-		[ "$number_cpus" -eq "$max_number_cpus" ] && break
+		if [[ "$number_cpus" =~ ^[0-9]$ ]]; then
```

Comment:
What if the system has 10 cpus? I'd extend the regex to allow that.

Reply:
True. I will change it to match multiple digits.

Reply:
Done